### DISCUSSION TITLE
Serve HLS Video from directus stored in S3/Minio bucket

### DISCUSSION THREAD

**Discussion by navanshu** [directus/directus#17825] [Archive]
Hi, I am using directus for a website as a CMS and want to provide users the ability to stream video files for a smooth playback.
Is it possible maybe with the combined use of [Nginx MP4 module](https://nginx.org/en/docs/http/ngx_http_mp4_module.html)?

I can also do pre conversion of those files and store it in a different bucket though. But if they can be converted on fly that would be much simpler.

**Comment by M-GRV** [comment]
Maybe a MUX extension could be made when I have time to do so :)
